compaction/compaction.cc

```diff
@@ -76,6 +76,7 @@
 #include "mutation_fragment_stream_validator.hh"
 #include "utils/UUID_gen.hh"
 #include "utils/utf8.hh"
+#include "utils/fmt-compat.hh"
```

Comment:
I don't see this new header in the commit.

Reply:
oops!

Reply:
Corrected.